.gitignore

```diff
@@ -1,9 +1,4 @@
+build
+prebuilds
```

Comment:
you can't gitignore prebuilds, then npm wont publish them

Reply:
npm publishes everything in `files` in `package.json` even if `.gitignored`'ed.

Reply:
oh really, then ok